packages/ra-ui-materialui/src/detail/Create.tsx

```diff
@@ -54,9 +54,23 @@ export const Create = <RecordType extends RaRecord = any>(
     props: CreateProps<RecordType> & { children: ReactElement }
 ): ReactElement => {
     useCheckMinimumRequiredProps('Create', ['children'], props);
+    const {
+        resource,
```

Comment:
disableAuthentication, hasEdit, hasShow are missing

Reply:
I omitted them because none of these are declared in the type of `props`

```
Property 'disableAuthentication' does not exist on type 'CreateProps<RecordType> & { children: ReactElement<any, string | JSXElementConstructor<any>>; }'
```

Should I add them?

Reply:
yes, please

Reply:
done